src/DevTools/createTestEnv.tsx

```diff
@@ -0,0 +1,203 @@
+import { ConnectedModalDialog } from "Apps/Order/Dialogs"
+import { createMockFetchQuery, MockBoot, renderRelayTree } from "DevTools"
+import React, { ReactElement } from "react"
+import { GraphQLTaggedNode } from "react-relay"
+import { Network } from "relay-runtime"
+import { Breakpoint } from "Utils/Responsive"
+import { RootTestPage } from "./RootTestPage"
+
```

Comment:
It would be helpful if each of these heavier infra-like classes had a description at the top describing purpose and perhaps some example blocks: 

```
@example 
... code 
```